README.md

```diff
@@ -1,2 +1,21 @@
-# o11y
-Stonesoup observability 
+# RHTAP Observability 
+
+This repository contains Prometheus recording rule files and alert rule files for observability use, as well as unit tests for these rules and alerts.
+
+## Recording Rules
+
+- `appstudio_container_resource_limits` collects all the resources limits (cpu and memory) for all containers and init_containers, and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric
+
+- `appstudio_container_resource_minutes_gauge` uses the `appstudio_container_resource_limits` to create a 1 minutes gauge and calculate the limits according to the live time of the container
+
+- `appstudio_container_network_transmit_bytes_total` collects the network egress of containers and adding the `label_pipelines_appstudio_openshift_io_type` label to our metric.
+
+### Limitations
+
+`appstudio_container_network_transmit_bytes_total` is using cAdvisor's `container_network_transmit_bytes_total` to collect the data.  
+We encounetered several issues while testing it with pods containing init_containers that transmit data.   
```

Comment:
Did we resolve the issues we encountered? If so, should it be mentioned?

Reply:
I think we should just mention the limitations rather than explaining the process we had.

Reply:
Done, I added the description of the limitation